samples/v1/test/analyzing_sentiment.test.yaml

```diff
@@ -0,0 +1,74 @@
+type: test/samples
```

Comment:
I'm a little surprised to see all the yamlz checked in here.  I would have guessed this is content used to generate the samples, and would live in an external repository.  Should we have these here?

Reply:
agreed, any chance we could grab this externally, perhaps during the `synthtool` step?